src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,10 +10,40 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class FileWorkItemsOptions
     {
         [Option(
-            'f',
+            't',
             "filing-target-type",
             HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
             Required = true)]
-        public string TargetType { get; internal set; }
+        public string FilingTargetType { get; internal set; }
+
+        [Option(
+            'p',
+            "project-uri",
+            HelpText = "The absolute URI of the project under which the work items are to be filed.",
```

Comment:
>The absolute URI of the project under which the work items are to be filed." [](start = 24, length = 76)

it might be useful to provide an example here? e.g. [https://dev.azure.com/{org}/{project](https://dev.azure.com/{org}/{project)} or [https://github.com/{org}/{project](https://github.com/{org}/{project)}

Reply:
Good idea.

---
In reply to: [306566043](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306566043) [](ancestors = 306566043)

Reply:
Fixed.

---
In reply to: [306569470](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306569470) [](ancestors = 306569470,306566043)